Sources/ViewModels/ALKConversationViewModel.swift

```diff
@@ -771,6 +782,86 @@ open class ALKConversationViewModel: NSObject, Localizable {
         }
     }
     
+    func updateMetaDataForCustomField(message : ALMessage){
+        if let lastMessage = alMessages.last?.messageModel,
+           let replyMetaData = lastMessage.getReplyMetaData() {
+            message.metadata.addEntries(from: replyMetaData)
+        }
+    }
+    
+    func updateUser(kmField: KMField, message: String) {
```

Comment:
Why are we naming it as message:String . Are we passing message here

Reply:
yes the message that the user sent we are passing that to update the userDetails